exporter/graph/graph_crawler.py

```diff
@@ -33,7 +33,13 @@ def generate_experiment_graph(self, process: MetadataResource, project: Metadata
         return experiment_process_graph.extend(supplementary_files_graph)
 
     def experiment_graph_for_process(self, process: MetadataResource) -> ExperimentGraph:
-        return self._crawl(process)
+        upper_graph = self._crawl_up(process)
```

Comment:
Why do you use "upper" and "lower" instead of "inputs" and"derived"?

Reply:
That's a good suggestion!